packages/core/admin/admin/src/content-manager/components/Inputs/utils/fieldDescription.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+/**
+ * Constructs a suitable description, taking into account a fields minimum and
+ * maximum length
+ *
+ * @param {String} description - the fields description
+ * @param {Number} minLength - the minimum length of the field
+ * @param {Number} maxLength -  the maximum length of the field
+ * @returns
+ */
+const buildDescription = (description, minLength, maxLength) => {
+  const minMaxDescription = [];
+
+  if (minLength) {
+    minMaxDescription.push(`min. ${minLength}`);
+  }
+  if (minLength && maxLength) {
+    minMaxDescription.push(`/`);
+  }
+  if (maxLength) {
+    minMaxDescription.push(`max. ${maxLength}`);
+  }
+  if (minMaxDescription.length > 0) {
+    minMaxDescription.push(`characters{br}`);
+  }
+
+  return {
+    id: description,
+    defaultMessage: `${minMaxDescription.join(' ')}${description}`,
+    values: {
+      br: <br />,
+    },
+  };
+};
```

Comment:
If you define everything together it means it becomes much harder to the user to localize the translation of a field, because it now contains also min/ max values which might change dynamically.

What do you think about passing mix and max in as values? I think this is more compatible with the intl API:

```suggestion
const buildDescription = (description, minLength, maxLength) => {
  const minMaxDescription = [];

  if (minLength) {
    minMaxDescription.push(`min. {minLength}`);
  }

  if (maxLength) {
    minMaxDescription.push(`max. {maxLength}`);
  }

  return {
    id: description,
    defaultMessage: `${minMaxDescription.join('/')} characters{br} ${description}`,
    values: {
      minLength,
      maxLength,
      br: <br />,
    },
  };
};
```

I'm still thinking whether it would be better to split the two things up and have one translation string for min/max and one for description. You could then concatenate them when you build the prop:


```js
description={`formatMessage(minMax) formatMessage(description)`}
```

I think I'd like this more because then users can handle this more granular and we can add one string to our translation files which handles only min and max.

What do you think?